src/components/ContactsPage.js

```diff
@@ -29,6 +37,31 @@ class ContactsPage extends React.Component {
     );
   }
 
+  changeIsChecked(id) {
+    this.setState((prevState) => {
+      const contacts = [...prevState.contacts];
+      const contact = contacts.find(item => item.id === id);
+      contact.isChecked = !contact.isChecked;
+      return { contacts };
+    });
+  }
+  
+  increaseLikes(id) {
+    this.setState((prevState) => {
+      const contacts = [...prevState.contacts];
+      contacts.find(item => item.id === id).likes += 1;
+      return { contacts };
+    });
+  }
+
+  decreaseLikes(id) {
+    this.setState((prevState) => {
+      const contacts = [...prevState.contacts];
+      contacts.find(item => item.id === id).likes -= 1;
+      return { contacts };
+    });
+  }
+
   render() {
     return this.getAppropriateJSX;
```

Comment:
Remove JSX from any naming in this file. It's obvious that those functions return JSX. Name them more specific to the contactsPage. Rename this to something like getContactsPage or even better renderContactsPage.